app/views/questionnaire.py

```diff
@@ -294,6 +294,7 @@ def submit_answers(eq_id, form_type, collection_id):
         path_finder = PathFinder(g.schema_json, answer_store, metadata)
         submitter = SubmitterFactory.get_submitter()
         message = convert_answers(metadata, g.schema_json, answer_store, path_finder.get_routing_path())
+        message['completed'] = True
```

Comment:
could this boolean be defaulted on convert_answers? that way if this step isn't done after then it will default.